sdk/appconfiguration/azure-appconfiguration-provider/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "python",
   "TagPrefix": "python/appconfiguration/azure-appconfiguration-provider",
-  "Tag": "python/appconfiguration/azure-appconfiguration-provider_51fb3fb738"
+  "Tag": "python/appconfiguration/azure-appconfiguration-provider_28c78fbf77"
```

Comment:
What is this change?

Reply:
The Python SDK does unit tests by replaying recorded responses. I had to update the recordings due to changes in the PR.